src/processors/processVrmRequest.ts

```diff
@@ -13,7 +13,7 @@ export const processPatchVrmRequest = (currentRecord: TechRecordGet, userDetails
     const updatedNewRecord = setCreatedAuditDetails(newRecord, userDetails.username, userDetails.msOid, new Date().toISOString(), currentRecord.techRecord_statusCode as StatusCode);
     (updatedNewRecord as TechRecordHgv | TechRecordMotorcycle | TechRecordCar | TechRecordPsv).primaryVrm = newVrm.toUpperCase();
     const updatedRecordToArchive = setLastUpdatedAuditDetails(recordToArchive, userDetails.username, userDetails.msOid, new Date().toISOString(), StatusCode.ARCHIVED);
-
+    newRecord.techRecord_reasonForCreation = 'Update VRM - Cherished Transfer';
     return [updatedRecordToArchive, newRecord];
```

Comment:
```suggestion
    return [updatedRecordToArchive, updatedNewRecord];
```

Reply:
Sorry I completely misread that. I'll sort it now